WordPress/Classes/ViewRelated/NUX/LoginEpilogueAnimator.swift

```diff
@@ -0,0 +1,80 @@
+import UIKit
+
+final class LoginEpilogueAnimator: NSObject, UIViewControllerAnimatedTransitioning {
+
+    private let loginAnimationDuration = 0.3
+    private let quickStartAnimationDuration = 0.3
+
+    func transitionDuration(using transitionContext: UIViewControllerContextTransitioning?) -> TimeInterval {
+        return loginAnimationDuration + quickStartAnimationDuration
+    }
+
+    func animateTransition(using transitionContext: UIViewControllerContextTransitioning) {
+        guard let loginEpilogueViewController = transitionContext.viewController(forKey: .from) as? LoginEpilogueViewController,
+              let quickStartPromptViewController = transitionContext.viewController(forKey: .to) as? QuickStartPromptViewController,
+              let selectedCell = loginEpilogueViewController.tableViewController?.selectedCell else {
+                  return
+              }
+
+        let containerView = transitionContext.containerView
+        containerView.backgroundColor = loginEpilogueViewController.view.backgroundColor
+
+        let cellSnapshot = selectedCell.contentView.snapshotView(afterScreenUpdates: false)
+        let cellSnapshotFrame = containerView.convert(selectedCell.contentView.frame, from: selectedCell)
+        cellSnapshot?.frame = cellSnapshotFrame
+        selectedCell.contentView.alpha = 0.0
+
+        let loginContainer = UIView(frame: loginEpilogueViewController.view.frame)
+        let loginSnapshot = loginEpilogueViewController.view.snapshotView(afterScreenUpdates: true)
+        loginContainer.backgroundColor = loginEpilogueViewController.view.backgroundColor
+
+        if let loginSnapshot = loginSnapshot {
+            loginSnapshot.layer.anchorPoint = CGPoint(x: 0.5, y: cellSnapshotFrame.origin.y / loginContainer.bounds.height)
+            loginSnapshot.frame = loginContainer.frame
+            loginContainer.addSubview(loginSnapshot)
+            containerView.addSubview(loginContainer)
+        }
+
+        if let cellSnapshot = cellSnapshot {
+            containerView.addSubview(cellSnapshot)
+        }
+
+        quickStartPromptViewController.view.alpha = 0.0
+        let safeAreaTop = loginEpilogueViewController.view.safeAreaInsets.top
+        quickStartPromptViewController.scrollViewTopVerticalConstraint.constant = cellSnapshotFrame.origin.y - safeAreaTop
+        quickStartPromptViewController.promptTitleLabel.alpha = 0.2
+        quickStartPromptViewController.promptDescriptionLabel.alpha = 0.2
+        quickStartPromptViewController.showMeAroundButton.alpha = 0.0
+        quickStartPromptViewController.noThanksButton.alpha = 0.0
```

Comment:
Same for these.

Reply:
These were also moved to the constant enum.